src/components/editor/metaform-editor.tsx

```diff
@@ -265,9 +265,12 @@ const MetaformEditor: React.FC<Props> = ({
             selected={ selected }
             onDeleteClick={ onFieldDeleteClick(sectionIndex, fieldIndex) }
           >
-            <Typography sx={{ color: "gray", margin: "5px" }}>
+            <Typography sx={{ color: "gray", margin: "0px 5px 0px 0px" }}>
```

Comment:
Instead of explicitly typing all sides for margin you could `marginRight: "5px"` or with the _sx_ prop `mr: "5px"`